### PR TITLE
WD-12247 Dev Create /data/mongodb/support

### PR DESCRIPTION
**Pull request by immortalcodes**
## Done

Created /data/mongodb/support
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Perform the checks for the new-webpage.
- [Design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=3-8106&t=Y4hBK1I6bTimfqDu-0)
- [Copydoc](https://docs.google.com/document/d/1XjFTw--fDl7ZpI6LI--UBKkAV4Yiz-gEoo2gcLuVdeA/edit)
- [Demo](https://canonical-com-1340.demos.haus/data/mongodb/support)

## Issue / Card

Fixes # [WD-12247](https://warthogs.atlassian.net/browse/WD-12247)

## Screenshots

[if relevant, include a screenshot]

[WD-12247]: https://warthogs.atlassian.net/browse/WD-12247?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by immortalcodes** [comment]
> @immortalcodes Please add proper demo link to PR description QA steps.

Hi, the demo link works fine for me, could you please give more details about the problem you are facing

File: templates/data/mongodb/support.html [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+  </section>
+
+  <section class="p-section">
+    <div class="row">
+      <hr />
+      <div class="col-3 col-medium-2">
+        <h2 class="p-text--small-caps">Why choose Canonical for MongoDB support</h2>
+      </div>
+      <div class="col-9 col-medium-4">
+        <ul class="p-list--divided">
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply templates/data/mongodb/support.html:58]: Yes, this is part of the design file.

File: templates/data/mongodb/support.html [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
+      <hr />
+      <div class="col">
+        <h2>Managed services</h2>
+      </div>
+    </div>
```
Comment (mtruj013) [inline templates/data/mongodb/support.html:147]: Why a 50-50 row that is not intended to be used as such? The same can be achieved like so: 
```suggestion
    <div class="u-fixed-width p-section--shallow">
      <hr />
      <h2>Managed services</h2>
    </div>
```

File: templates/data/mongodb/support.html [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+      <hr />
+      <div class="col p-section">
+        <h2>Come for MongoDB support, access all Canonical solutions</h2>
+      </div>
+      <div class="col p-section">
+        <p>
+          With Ubuntu Pro + Support for MongoDB, you get an all-access pass to everything Canonical has to offer, including best-in-class open source solutions for enterprise data management and AI. Ubuntu Pro subscriptions are competitively priced per server, not per application.
+        </p>
+      </div>
+      <div class="p-logo-section--dense">
```
Comment (mtruj013) [inline templates/data/mongodb/support.html:261]: These logos look extremely small, did design confirm the sizing here?
image

File: templates/data/mongodb/support.html [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="is-muted" />
+            <p>
+              <a href="/data/kafka">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/662c72e2-mongodb.png",
```
Comment (mtruj013) [inline templates/data/mongodb/support.html:334]: MongoDB logo looks like it's just falling back to it's alt text even though it isn't. Left a [comment](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl?node-id=3-8107#911214640) on the figma

File: templates/data/mongodb/support.html [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Learn more about Canonical's MongoDB support</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/mongodb-for-enterprises">WatchMongoDB for modern data management</a>
```
Comment (mtruj013) [inline templates/data/mongodb/support.html:389]: Typo on the doc
```suggestion
            <a href="https://ubuntu.com/engage/mongodb-for-enterprises">Watch MongoDB for modern data management</a>
```

File: templates/data/mongodb/support.html [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+        <div class="p-section--shallow">
+          <p class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/simplifying-mongodb-operations">Watch how to simplify MongoDB operations</a>
+          </p>
+          <p>Navigate the complexity of managing your database with Charmed MongoDB</p>
+        </div>
+        <hr class="is-muted" />
+        <div class="p-section--shallow">
+          <p class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/mongodb-support-security">Read the MongoDB® Toolkit</a>
```
Comment (mtruj013) [inline templates/data/mongodb/support.html:403]:
```suggestion
            <a href="https://ubuntu.com/engage/mongodb-support-security">Read the MongoDB<sup>&reg;</sup> Toolkit</a>
```

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Thanks @immortalcodes, some comments in the code

File: templates/data/mongodb/support.html [hunk 7]
```diff
@@ -0,0 +1,429 @@
...
+                        hi_def=True,
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="col-9 col-medium-4">
+          <div class="p-section--shallow">
+            <h1>MongoDB support</h1>
+            <p>
+              Draw on Canonical's expertise and run MongoDB securely and efficiently in your enterprise. Get 24/7 phone/ticket support for your critical data infrastructure with <a href="https://ubuntu.com/pro">Ubuntu Pro</a> + Support.
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply templates/data/mongodb/support.html:39]: Yeah I know, in my last PRs Ant pointed out this as suggestion, to link the /pro page.
I can omit it it as well

File: templates/data/mongodb/support.html (hunk 3 above)
Comment (immortalcodes) [reply templates/data/mongodb/support.html:261]: Lyubo suggested me to do this way [here](https://canonical.com/data/opensearch/support). This has to be corrected in Vanilla, next version should fix this.

File: templates/data/mongodb/support.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/data/mongodb/support.html:38]: I leave that decision to you, but please keep in mind that docs are the source of truth for content. Whatever content changes you make need to be reflected there

**Review by mtruj013 — APPROVED** [review]

**Pull request merged**